DuckDuckGo/HomePage/View/PromotionView.swift

```diff
@@ -85,8 +84,30 @@ extension HomePage.Views {
             }
         }
 
-        private var text: some View {
-            Text(viewModel.text)
+        private var textContent: some View {
+            VStack(alignment: .leading, spacing: viewModel.title == nil ? 0 : 8) {
+                title
+                description.foregroundColor(Color(.greyText))
+            }
+        }
+
+        private var title: some View {
+            Group {
+                if let title = viewModel.title {
+                    Text(title)
+                        .font(.system(size: 13).bold())
+                } else {
+                    EmptyView()
+                }
+            }
+       }
+
+        private var description: some View {
+            if let additionalBoldedDescription = viewModel.additionalBoldedDescription {
+                return Text(viewModel.description) + Text(additionalBoldedDescription).bold()
```

Comment:
This is probably OK for now but will need to be revisited as soon as we want to translate Privacy Pro.

I'd consider using Markdown for this string on macOS 12.0 and above, and on macOS 11 stick to plain text without the bold part.

One other thing I would suggest is so that we used `Text(verbatim:)` for strings that we don't want to translate (yet). This is mentioned in [macOS: Localization / translation guidelines](https://app.asana.com/0/904401899170/1206727265537758/f)